### DISCUSSION TITLE
WAF - 拦截地图（30日）不显示地域

### DISCUSSION THREAD

**Discussion by mjsoftking** [1Panel-dev/1Panel#6151] [Q&A]
### 1Panel 版本

v1.10.13-lts

### 问题描述

![image](https://github.com/user-attachments/assets/76c25677-4987-4784-a6df-ec0f6f771631)

### 重现步骤

`WAF - 拦截记录 - 清空日志`
执行`清空日志`后，之后的统计都不在地图上显示

**Comment by zhengkunwang223** [comment]
看一下这个 https://bbs.fit2cloud.com/t/topic/7004
或者等下个版本更新

**Comment by mjsoftking** [comment]
修复了就行，不急，等更新了
搜索issues没找到关于地图的问题，所以提了一个
